### PR TITLE
first daft release

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/daft/daft.0.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  [make]
+]
+install: [
+  ["cp" "-t" bin "daft" "daft_ds" "daft_mds"]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/daft" "%{bin}%/daft_ds" "%{bin}%/daft_mds"]
+]
+depends: [
+  "obuild"
```
Comment (dsheets) [inline packages/daft/daft.0.0.2/opam]: This is probably a `{build}` dependency unless you extend `obuild` and so require to link with it at run-time.

**Comment by dsheets** [comment]
It looks like there is an issue with zmq on Linux and `cp -t` is a GNUism so that fails on OS X (and probably the BSDs). I think you can simply use `cp src1 src2 src3 target_dir` for the `cp` issue. The zmq issue may be related to the version of the library installed and a post-install failure message regarding the minimum version of the expected package in, e.g., the `zmq` package would be appreciated.

**Comment by UnixJunkie** [comment]
@dsheets thanks for the info, I'll try to fix all that today

**Comment by UnixJunkie** [comment]
@dsheets still some quite strange errors. I see that opam still works so well with a system-provided compiler ...

**Comment by UnixJunkie** [comment]
I am not sure I can do anything else to push this further in travis. It installs on my machine, even with the latest zmq ...

**Comment by UnixJunkie** [comment]
I think this pull request can be accepted: problems are with zmq, not with daft.

**Comment by samoht** [comment]
done

**Comment by UnixJunkie** [comment]
thanks!

**Pull request merged**